### ISSUE TITLE
`npm link xrpl` doesn't work properly

### ISSUE THREAD

**Issue by mvadari** [XRPLF/xrpl.js#2704]
Reproduction:
* Run `npm i -g xrpl`
* Then try running a script with `const xrpl = require("xrpl")`
* It fails with a `MODULE_NOT_FOUND` error
* Try then running `npm link xrpl` to link the global module in locally
* You get the following error:
```
npm error code 127
npm error path /usr/local/lib/node_modules/xrpl
npm error command failed
npm error command sh -c copyfiles ../../README.md xrpl/README.md
npm error sh: 1: copyfiles: not found
```

I believe this is because the `prepare` script is running - that line was added here: https://github.com/XRPLF/xrpl.js/pull/2530

**Comment by ckeshava** [comment]
will it help if you install the `copyfiles` package? `npm install copyfiles -g` ?

**Comment by mvadari** [comment]
> will it help if you install the `copyfiles` package? `npm install copyfiles -g` ?

Yes, but I shouldn't need to install a separate package to be able to run `npm link xrpl`.

I don't think just adding `copyfiles` to the normal, non-dev dependencies is the answer either, but I don't understand `npm link` well enough to understand what's happening here.
